### ISSUE TITLE
WCAG compliancy

### ISSUE THREAD

**Issue by sandrauddman** [sandrauddman/agileProjectGrupp2#22]
make sure that the diffrent parts of the page follows WCAG 2.0 AA guidelines.  

-missing form label on search
-go through semantic html
-low contrast on darkmode search form (optional) 
-thumbnail alt text redundant? (optional)
